lib/Frontend/Frontend.cpp

```diff
@@ -256,9 +256,9 @@ bool CompilerInstance::setUpASTContextIfNeeded() {
 }
 
 void CompilerInstance::setupStatsReporter() {
-  const auto &Invok = getInvocation();
+  const auto &Invoke = getInvocation();
```

Comment:
As noted in the parent PR, I'm happy to drop `invoke` changes (or anything else). I'm not sure if there's a specific reason that the `e` was omitted (it's pretty consistent, but it will appear in a couple of parallel PRs).

Reply:
It's not meant to be the verb `invoke`, but the `k` is also weird I agree.

Reply:
This is fine.